components/802.15.4_RF/stm-s2lp-rf-driver/README.md

```diff
@@ -3,4 +3,9 @@
 Support for:
  * S2-LP
 
-This driver is used with 6LoWPAN and Wi-SUN stacks.
\ No newline at end of file
+This driver is used with 6LoWPAN and Wi-SUN stacks.
+
+Driver is tested with X-Nucleo-S2868A1 RF expansion board. https://www.st.com/en/ecosystems/x-nucleo-s2868a1.html
+
+NOTE: Default SPI SCLK pin configuration is D13. On X-Nucleo-S2868A1, resistor R6 must be soldered instead of R11. For more information, see user manual UM2405. https://www.st.com/resource/en/user_manual/dm00498153.pdf
```

Comment:
Could be turned into a [link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) for readability - [UM2405 User manual](https://www.st.com/resource/en/user_manual/dm00498153.pdf)

Reply:
OK, we will keep that in mind when updating the readme next time.